Update Faker-js version to 7.6.0 in package.json of codecept currently it is 5.5.3 and it is having some issue with typescript.

#### What are you trying to achieve?
Update Faker-js version to 7.6.0 in package.json of codecept currently it is 5.5.3 and it is having some issue with typescript.
#### What do you get instead?

> Provide console output if related. Use `--verbose` mode for more details.
Getting below error :
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: XXX-codecept@1.0.0
npm ERR! Found: @faker-js/faker@7.6.0
npm ERR! node_modules/@faker-js/faker
npm ERR!   peer @faker-js/faker@"^7.6.0" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer @faker-js/faker@"^5.5.3" from codeceptjs@3.3.6
npm ERR! node_modules/codeceptjs
npm ERR!   codeceptjs@"^3.3.6" from the root project
npm ERR!   peer codeceptjs@">= 2.3.3" from @codeceptjs/configure@0.10.0
npm ERR!   node_modules/@codeceptjs/configure
npm ERR!     @codeceptjs/configure@"^0.10.0" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
```bash
# paste output here
```

> Provide test source code if related

```js
// paste test
```

### Details

* CodeceptJS version: 3.3.6
* NodeJS Version: 8.15.0
* Operating System: Mac OS
* puppeteer || webdriverio || testcafe version (if related) : Playwright.
* Configuration file:

```js
# paste config here
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -28,8 +28,8 @@
 
 📖 Documentation
 
-* Updated [Auickstart](https://codecept.io/quickstart/) with detailed explanation of questions in init
-* Added [Translation](/translations/) guide
+* Updated [Quickstart](https://codecept.io/quickstart/) with detailed explanation of questions in init
+* Added [Translation](/translation/) guide
 * Updated [TypeScript](https://bit.ly/3XIMq6n) guide for promise-based typings
 * Reordered guides list on a website
 

--- a/docs/best.md
+++ b/docs/best.md
@@ -215,7 +215,7 @@ include: {
 * When you need to customize access to API and go beyond what ApiDataFactory provides, implement DAO:
 
 ```js
-const faker = require('@faker-js/faker');
+const { faker } = require('@faker-js/faker');
 const { I } = inject();
 const { output } = require('codeceptjs');
 

--- a/docs/changelog.md
+++ b/docs/changelog.md
@@ -7,6 +7,69 @@ layout: Section
 
 # Releases
 
+## 3.3.7
+
+🛩️ Features
+
+* **Promise-based typings** for TypeScript definitions in [#3465](https://github.com/codeceptjs/CodeceptJS/issues/3465) by **[nlespiaucq](https://github.com/nlespiaucq)**. If you use TypeScript or use linters [check how it may be useful to you](https://bit.ly/3XIMq6n).
+* **Translation** improved to use [custom vocabulary](https://codecept.io/translation/). 
+* **[Playwright]** Added methods in [#3398](https://github.com/codeceptjs/CodeceptJS/issues/3398) by **[mirao](https://github.com/mirao)**
+  * `restartBrowser` - to restart a browser (with different config)
+  * `_createContextPage` - to create a new browser context with a page from a helper
+* Added [Cucumber custom types](/bdd#custom-types) for BDD in [#3435](https://github.com/codeceptjs/CodeceptJS/issues/3435) by **[Likstern](https://github.com/Likstern)** 
+* Propose using JSONResponse helper when initializing project for API testing. [#3455](https://github.com/codeceptjs/CodeceptJS/issues/3455) by **[PeterNgTr](https://github.com/PeterNgTr)** 
+* When translation enabled, generate tests using localized aliases. By **[davertmik](https://github.com/davertmik)**
+* **[Appium]** Added `checkIfAppIsInstalled` in [#3507](https://github.com/codeceptjs/CodeceptJS/issues/3507) by **[PeterNgTr](https://github.com/PeterNgTr)** 
+
+🐛 Bugfixes
+
+* Fixed [#3462](https://github.com/codeceptjs/CodeceptJS/issues/3462) `TypeError: Cannot read properties of undefined (reading 'setStatus')` by **[dwentland24](https://github.com/dwentland24)** in [#3438](https://github.com/codeceptjs/CodeceptJS/issues/3438)
+* Fixed creating steps file for TypeScript setup [#3459](https://github.com/codeceptjs/CodeceptJS/issues/3459) by **[PeterNgTr](https://github.com/PeterNgTr)** 
+* Fixed issue of after all event in `run-rerun` command after complete execution [#3464](https://github.com/codeceptjs/CodeceptJS/issues/3464) by **[jain-neeeraj](https://github.com/jain-neeeraj)** 
+* [Playwright][WebDriver][Appium] Do not change `waitForTimeout` value on validation. See [#3478](https://github.com/codeceptjs/CodeceptJS/issues/3478) by **[pmajewski24](https://github.com/pmajewski24)**. Fixes [#2589](https://github.com/codeceptjs/CodeceptJS/issues/2589)
+* [Playwright][WebDriver][Protractor][Puppeteer][TestCafe] Fixes `Element "{null: undefined}" was not found` and `element ([object Object]) still not present` messages when using object locators. See [#3501](https://github.com/codeceptjs/CodeceptJS/issues/3501) and [#3502](https://github.com/codeceptjs/CodeceptJS/issues/3502) by **[pmajewski24](https://github.com/pmajewski24)**
+* **[Playwright]** Improved file names when downloading file in [#3449](https://github.com/codeceptjs/CodeceptJS/issues/3449) by **[PeterNgTr](https://github.com/PeterNgTr)**. Fixes [#3412](https://github.com/codeceptjs/CodeceptJS/issues/3412) and [#3409](https://github.com/codeceptjs/CodeceptJS/issues/3409)
+* Add default value to `profile` env variable. See [#3443](https://github.com/codeceptjs/CodeceptJS/issues/3443) by **[dwentland24](https://github.com/dwentland24)**. Resolves [#3339](https://github.com/codeceptjs/CodeceptJS/issues/3339)
+* **[Playwright]** Using system-native path separator when saving artifacts in [#3460](https://github.com/codeceptjs/CodeceptJS/issues/3460) by **[PeterNgTr](https://github.com/PeterNgTr)**
+* **[Playwright]** Saving videos and traces from multiple sessions in [#3505](https://github.com/codeceptjs/CodeceptJS/issues/3505) by **[davertmik](https://github.com/davertmik)**
+* **[Playwright]** Fixed `amOnPage` to navigate to `about:blank` by **[zaxoavoki](https://github.com/zaxoavoki)** in [#3470](https://github.com/codeceptjs/CodeceptJS/issues/3470) Fixes [#2311](https://github.com/codeceptjs/CodeceptJS/issues/2311)
+* Various typing improvements by **[AWolf81](https://github.com/AWolf81)** **[PeterNgTr](https://github.com/PeterNgTr)** **[mirao](https://github.com/mirao)**  
+
+📖 Documentation
+
+* Updated [Auickstart](https://codecept.io/quickstart/) with detailed explanation of questions in init
+* Added [Translation](/translations/) guide
+* Updated [TypeScript](https://bit.ly/3XIMq6n) guide for promise-based typings
+* Reordered guides list on a website
+
+## 3.3.6
+
+* [`run-rerun`](https://codecept.io/commands/#run-rerun) command was re-introduced by **[dwentland24](https://github.com/dwentland24)** in [#3436](https://github.com/codeceptjs/CodeceptJS/issues/3436). Use it to perform run multiple times and detect flaky tests
+* Enabled `retryFailedStep` by default in `@codeceptjs/configure` v 0.10. See https://github.com/codeceptjs/configure/pull/26
+* **[Playwright]** Fixed properties types "waitForNavigation" and "firefox" by **[mirao](https://github.com/mirao)** in [#3401](https://github.com/codeceptjs/CodeceptJS/issues/3401)
+* **[REST]** Changed "endpoint" to optional by **[mirao](https://github.com/mirao)** in [#3404](https://github.com/codeceptjs/CodeceptJS/issues/3404)
+* **[REST]** Use [`secret`](/secrets) for form encoded string by **[PeterNgTr](https://github.com/PeterNgTr)**:
+
+```js
+const secretData = secret('name=john&password=123456');
+const response = await I.sendPostRequest('/user', secretData);
+```
+
+* [Playwright]Fixed docs related to fixed properties types "waitForNavigation" and "firefox" by **[mirao](https://github.com/mirao)** in [#3407](https://github.com/codeceptjs/CodeceptJS/issues/3407)
+* [Playwright]Fixed parameters of startActivity() by **[mirao](https://github.com/mirao)** in [#3408](https://github.com/codeceptjs/CodeceptJS/issues/3408)
+* Move semver to prod dependencies by **[timja](https://github.com/timja)** in [#3413](https://github.com/codeceptjs/CodeceptJS/issues/3413)
+* check if browser is W3C instead of Android by **[mikk150](https://github.com/mikk150)** in [#3414](https://github.com/codeceptjs/CodeceptJS/issues/3414)
+* Pass service configs with options and caps as array for browsers… by **[07souravkunda](https://github.com/07souravkunda)** in [#3418](https://github.com/codeceptjs/CodeceptJS/issues/3418)
+* fix for type of "webdriver.port" by **[ngraf](https://github.com/ngraf)** in [#3421](https://github.com/codeceptjs/CodeceptJS/issues/3421)
+* fix for type of "webdriver.smartWait" by **[pmajewski24](https://github.com/pmajewski24)** in [#3426](https://github.com/codeceptjs/CodeceptJS/issues/3426)
+* fix(datatable): mask secret text by **[PeterNgTr](https://github.com/PeterNgTr)** in [#3432](https://github.com/codeceptjs/CodeceptJS/issues/3432)
+* fix(playwright) - video name and missing type by **[PeterNgTr](https://github.com/PeterNgTr)** in [#3430](https://github.com/codeceptjs/CodeceptJS/issues/3430)
+* fix for expected type of "bootstrap", "teardown", "bootstrapAll" and "teardownAll" by **[ngraf](https://github.com/ngraf)** in [#3424](https://github.com/codeceptjs/CodeceptJS/issues/3424)
+* Improve generate pageobject `gpo` command to work with TypeScript by **[PeterNgTr](https://github.com/PeterNgTr)** in [#3411](https://github.com/codeceptjs/CodeceptJS/issues/3411)
+* Fixed dry-run to always return 0 code and exit 
+* Added minimal version notice for NodeJS >= 12
+* fix(utils): remove . of test title to avoid confusion by **[PeterNgTr](https://github.com/PeterNgTr)** in [#3431](https://github.com/codeceptjs/CodeceptJS/issues/3431)
+
 ## 3.3.5
 
 🛩️ Features
@@ -132,8 +195,6 @@ npx codeceptjs run test-dir/*"
 📖 Documentation
 * Improving the typings in many places
 * Improving the return type of helpers for TS users ([#3245](https://github.com/codeceptjs/CodeceptJS/issues/3245)) - **[nlespiaucq](https://github.com/nlespiaucq)**
-
-🇺🇦 Added #StandWithUkraine hashtag notice to CLI
 
 ## 3.3.0
 

--- a/docs/helpers/ApiDataFactory.md
+++ b/docs/helpers/ApiDataFactory.md
@@ -53,8 +53,8 @@ See the example for Posts factories:
 ```js
 // tests/factories/posts.js
 
-const Factory = require('rosie').Factory;
-const faker = require('@faker-js/faker');
+const { Factory } = require('rosie');
+const { faker } = require('@faker-js/faker');
 
 module.exports = new Factory()
    // no need to set id, it will be set by REST API

--- a/docs/helpers/GraphQLDataFactory.md
+++ b/docs/helpers/GraphQLDataFactory.md
@@ -53,8 +53,8 @@ See the example for Users factories:
 ```js
 // tests/factories/users.js
 
-const Factory = require('rosie').Factory;
-const faker = require('@faker-js/faker');
+const { Factory } = require('rosie').Factory;
+const { faker } = require('@faker-js/faker');
 
 // Used with a constructor function passed to Factory, so that the final build
 // object matches the necessary pattern to be sent as the variables object.

--- a/docs/index.md
+++ b/docs/index.md
@@ -86,7 +86,7 @@ Scenario('Checkout test', ({ I }) => {
 Can we use it for long scenarios? Sure!
 
 ```js
-const faker = require('@faker-js/faker');                               // Use 3rd-party JS code
+const { faker } = require('@faker-js/faker');                               // Use 3rd-party JS code
 
 Feature('Store');
 

--- a/lib/helper/ApiDataFactory.js
+++ b/lib/helper/ApiDataFactory.js
@@ -46,8 +46,8 @@ const REST = require('./REST');
  * ```js
  * // tests/factories/posts.js
  *
- * const Factory = require('rosie').Factory;
- * const faker = require('@faker-js/faker');
+ * const { Factory } = require('rosie');
+ * const { faker } = require('@faker-js/faker');
  *
  * module.exports = new Factory()
  *    // no need to set id, it will be set by REST API

--- a/lib/helper/GraphQLDataFactory.js
+++ b/lib/helper/GraphQLDataFactory.js
@@ -46,8 +46,8 @@ const GraphQL = require('./GraphQL');
  * ```js
  * // tests/factories/users.js
  *
- * const Factory = require('rosie').Factory;
- * const faker = require('@faker-js/faker');
+ * const { Factory } = require('rosie').Factory;
+ * const { faker } = require('@faker-js/faker');
  *
  * // Used with a constructor function passed to Factory, so that the final build
  * // object matches the necessary pattern to be sent as the variables object.

--- a/lib/plugin/fakerTransform.js
+++ b/lib/plugin/fakerTransform.js
@@ -1,4 +1,4 @@
-const faker = require('@faker-js/faker');
+const { faker } = require('@faker-js/faker');
 const transform = require('../transform');
 
 /**

--- a/lib/plugin/fakerTransform.js
+++ b/lib/plugin/fakerTransform.js
@@ -44,7 +44,7 @@ const transform = require('../transform');
 module.exports = function (config) {
   transform.addTransformerBeforeAll('gherkin.examples', (value) => {
     if (typeof value === 'string' && value.length > 0) {
-      return faker.fake(value);
+      return faker.helpers.fake(value);
     }
     return value;
   });

--- a/package.json
+++ b/package.json
@@ -94,7 +94,7 @@
   "devDependencies": {
     "@codeceptjs/detox-helper": "^1.0.2",
     "@codeceptjs/mock-request": "^0.3.1",
-    "@faker-js/faker": "^5.5.3",
+    "@faker-js/faker": "^7.6.0",
     "@pollyjs/adapter-puppeteer": "^5.1.0",
     "@pollyjs/core": "^5.1.0",
     "@types/inquirer": "^0.0.35",

--- a/test/data/graphql/users_factory.js
+++ b/test/data/graphql/users_factory.js
@@ -1,5 +1,5 @@
-const Factory = require('rosie').Factory;
-const faker = require('@faker-js/faker');
+const { Factory } = require('rosie');
+const { faker } = require('@faker-js/faker');
 
 module.exports = new Factory(function (buildObject) {
   this.input = { ...buildObject };

--- a/test/data/rest/posts_factory.js
+++ b/test/data/rest/posts_factory.js
@@ -1,5 +1,5 @@
-const Factory = require('rosie').Factory;
-const faker = require('@faker-js/faker');
+const { Factory } = require('rosie');
+const { faker } = require('@faker-js/faker');
 
 module.exports = new Factory()
   .attr('author', () => faker.name.findName())

